### PR TITLE
Update DevKit.md

### PR DESCRIPTION
**Pull request by jackbielinski**
Added installation instructions for VS2022

### CODE REVIEW

**Comment by zarlo** [comment]
you should merge it in to the VS2022 branch (https://github.com/CosmosOS/Cosmos/pull/2009)

File: Docs/articles/Installation/DevKit.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
    * You must clone the repository using Git. For a detailed walkthrough, [see here](https://help.github.com/articles/fork-a-repo/).
    * When following the tutorial, replace *OctoCat* with *CosmosOS* and *Spoon-Knife* with *Cosmos*.
```
Comment (ptobuon) [inline Docs/articles/Installation/DevKit.md:6]: This can be removed.

**Review by ptobuon — CHANGES_REQUESTED** [review]
There is only a suggested change. The rest is good.

File: Docs/articles/Installation/DevKit.md [hunk 2]
```diff
@@ -25,3 +26,10 @@ The `install-VS2019.bat` accepts the following parameters :
 - `-NOVSLAUNCH` Don't launch VS after installation.
 - `-IGNOREVS` Ignore running VS during installation.
 - `-VSEXPHIVE` or `/VSEXPHIVE` Use Visual Studio Experimental Hive for installation.
+
+## Installation for Visual Studio 2022
+Installing COSMOS with Visual Studio 2022 takes more effort as it is not fully supported, which is why it is recommended to download Visual Studio 2019 instead. **NOTE:** You can only install the DevKit for VS2022. Running `userkit install.bat` will use VS2019 instead.
+
+* [Download the DevKit on the VS2022 branch](https://github.com/CosmosOS/Cosmos/tree/VS2022)
+* Run `install-VS2022.bat` with admin privileges (UAC will ask for permission), needed for install in system directories.
+* When the installation is complete, Visual Studio will automatically open and you may begin programming with your new, modified copy of Cosmos.
```
Comment (MishaTY) [inline Docs/articles/Installation/DevKit.md:35]: This is no longer true, visual studio does not open after installation

File: Docs/articles/Installation/DevKit.md (hunk 1 above)
Comment (MishaTY) [reply Docs/articles/Installation/DevKit.md:6]: This can be removed

**Review by MishaTY — CHANGES_REQUESTED** [review]

**Comment by valentinbreiz** [comment]
Since VS2022 is merged in master, just use the master branch and install the devkit

**Pull request closed without merging**